todo_app/app.py

```diff
@@ -3,26 +3,38 @@
 import os
 
 from todo_app.flask_config import Config
-from todo_app.data.trello_items import get_items, add_item, change_item_list
+from todo_app.data.trello_items import Items
+from todo_app.models.view_model import ViewModel
 
-app = Flask(__name__)
-app.config.from_object(Config())
+def create_app():
+  app = Flask(__name__)
+  app.config.from_object(Config())
 
-@app.route('/')
-def index():
-    return render_template('index.html', tasks=get_items())
+  board_items = Items()
+  
+  @app.route('/')
+  def index():
+      item_view_model = ViewModel(board_items.items)
+      return render_template('index.html', view_model=item_view_model)
 
-@app.route('/add', methods=['POST'])
-def add():
-    add_item(request.form.get('task_title'))
+  @app.route('/add', methods=['POST'])
+  def add():
+      board_items.add_item(request.form.get('task_title'))
+      return redirect("/")
+
+  @app.route('/completed/<id>')
```

Comment:
For some reason I completely forgot that your to-dos are draggable to allow triggering this and spent a minute trying to find out why I'd not asked about this before - so this is mostly a note for future me to notice that this functionality _is_ implemented on the front end!

That said, it's always worth considering whether there's a UX improvement that could be made when a user gets confused - for example would making the boxes show [the cursor in the "grab" form](https://developer.mozilla.org/en-US/docs/Web/CSS/cursor) make this more obvious maybe?

Reply:
Apologies for the late response, didn't get any notifications for these comments! Good point about the grab cursor, it would definitely make it clearer, I'll look into that with the next refactor